samples/src/Microsoft.Azure.IIoT.App/src/Pages/Publishers.razor

```diff
@@ -34,7 +35,8 @@
         </tr>
     </thead>
     <tbody>
-        @foreach (var publisher in _pagedPublisherList.Results) {
+        @foreach (var publisher in _pagedPublisherList.Results)
```

Comment:
keep curly brace on the same line. This code style is now adopted in .blazor files